lib/emails.js

```diff
@@ -0,0 +1,14 @@
+class Emails {
+    constructor (client) {
+        this.client = client
+    }
+    sendTransactionalEmail (data, cb) {
+        return this.client._request({
+            method: 'POST',
+            path: '/email/public/v1/singleEmail/send',
+            body: data
+        }, cb)
+    }
+}
+
+module.exports = Emails;
```

Comment:
Would you mind adding a newline here? Here's a blog post describing why GitHub shows missing newlines at the end of files: https://robots.thoughtbot.com/no-newline-at-end-of-file